.mergify.yml

```
@@ -21,6 +21,9 @@ queue_rules:
               - label=proto:expect-breakage
     merge_conditions:
       - base=master
+      - or:
+          - label=bypass:fixup
+          - check-success=check-fixup-commits
```

Please add a comment that a merge condition is used to allow the PR to enter the queue in case the PR is already linear.

Also I would like to re-use the existing `bypass:linear-history` label instead. And if we add the conditions on the job as I suggested, we need to accept skipped checks as well

```suggestion
      # Rebase PRs with fixup commits are allowed to enter the merge queue but 
      # should not be allowed to merge if there are leftover fixup commits after rebase
      - or:
          - check-skipped=no-fixup-commits
          - check-success=no-fixup-commits
```